customer-service/src/main/java/de/senacor/innolab/graalvm/customer/service/CustomerService.java

```diff
@@ -16,6 +16,7 @@
 
     private CustomerRepository customerRepository;
 
+
```

Comment:
I add a newline to verify that the pipeline is triggered. I'll revert it once the PR is approved.